design/one-pager-package-format-v2.md

```diff
@@ -184,7 +176,7 @@ metadata:
   name: configuration-example
   # Optional. Must comply with Kubernetes API conventions. Annotations are
   # opaque to Crossplane, which will replicate them to the annotations of a
-  # PackageRevision when this package is unpacked, but otherwise ignore them.
+  # config revision when this package is unpacked, but otherwise ignore them.
```

Comment:
`Configuration` instead of `config` just for consistency

Reply:
I got lazy because my automatic 80 char line wrapping tool won't wrap comments inside YAML snippets for some reason. Fixed.